app/assets/javascripts/application.js

```diff
@@ -17,6 +17,9 @@ function openPopup(url, name, top, left) {
 
     $(function () {
 
+        var isSfr = window.location.pathname.indexOf('shortFormatPreSentenceReport') !== -1;
```

Comment:
For some reason it think these are changes (I did a rebase) - but my diff is showing no changes - can you check this looks ok?

Reply:
No idea why it's done this either - code looks fine though.